crates/matrix-sdk-sqlite/src/event_cache_store.rs

```diff
@@ -339,6 +339,16 @@ async fn run_migrations(conn: &SqliteAsyncConn, version: u8) -> Result<()> {
         .await?;
     }
 
+    if version < 5 {
+        conn.with_transaction(|txn| {
+            txn.execute_batch(include_str!(
+                "../migrations/event_cache_store/005_events_index_on_event_id.sql"
+            ))?;
+            txn.set_db_version(4)
```

Comment:
This should help with the tests :D

```suggestion
            txn.set_db_version(5)
```

Reply:
Glooups, thanks!